### DISCUSSION TITLE
Camera support surface pro

### DISCUSSION THREAD

**Discussion by midozalouk** [sebanc/brunch#1343] [Ideas]
Hello sir
Can you please add camera support to the surface price devices (esp surface pro 6)
The Linux kernel has added support to camera

If you don't have the required time can you please tell me how?

**Comment by tejasraman** [comment]
I’m not sebanc, and I have no idea about this, but an unstable build with kernel 5.15 was just released. Maybe try it out? 5.15 has cam support for your Surface.

[https://github.com/sebanc/brunch-unstable/releases/tag/r96-unstable-20220102](https://github.com/sebanc/brunch-unstable/releases/tag/r96-unstable-20220102)
